tests/data/short_circuit/three_phase/update_batch.json

```diff
@@ -2,7 +2,7 @@
   {
     "fault":
       [
-        {"id": 10, "fault_object": 1, "r": 0, "x": 0}
```

Comment:
don't we need cases with `nan` and/or `default_value` for `fault_phase`?

Reply:
Let us not include those in the validation cases since their results will be just a duplicate and crowd the already big cases. We can keep the validation cases only to theoretical validation of the algorithm. 

I can add default value as a case in unit tests instead.